docs/copied-from-beats/docs/security/api-keys.asciidoc

```diff
@@ -19,71 +19,114 @@ IMPORTANT: Review <<feature-roles>> before creating API keys for {beatname_uc}.
 [float]
 [[beats-api-key-publish]]
 === Create an API key for publishing
-To create an API key to use for writing data to {es}, use the
-{ref}/security-api-create-api-key.html[Create API key API], for example:
 
-[source,console,subs="attributes,callouts"]
-------------------------------------------------------------
-POST /_security/api_key
+In {kib}, navigate to **Stack Management** > **API keys** and click **Create API key**.
+
+[role="screenshot"]
+image::images/server-api-key-create.png[API key creation]
+
+Enter a name for your API key and select **Restrict privileges**.
+In the role descriptors box, assign the appropriate privileges to the new API key.
+For example:
+
+[source,json,subs="attributes,callouts"]
+----
 {
-  "name": "{beat_default_index_prefix}_host001", <1>
-  "role_descriptors": {
-    "{beat_default_index_prefix}_writer": { <2>
-      "cluster": ["monitor", "read_ilm"],
-      "index": [
-        {
-          "names": ["{beat_default_index_prefix}-*"],
-          "privileges": ["view_index_metadata", "create_doc"]
-        }
+    "{beat_default_index_prefix}_writer": {
+        "cluster": ["monitor", "read_ilm"], <1>
+        "index": [ <2>
+            {
+                "names": ["{beat_default_index_prefix}-*"],
+                "privileges": ["view_index_metadata", "create_doc"]
+            }
```

Comment:
Question for reviewers. Are these permissions correct for a **publishing** user? I didn't change anything here, but I did notice that our [**Grant privileges and roles needed for writing events**](https://www.elastic.co/guide/en/apm/server/current/privileges-to-publish-events.html) documentation shows different req'd privs.

Reply:
Good catch! The linked docs have a NOTE at the beginning that the `setup` role is separated from the `writer` role to minimize the privileges required during writing. That's why the dedicated writer role has less privileges. But even with that, there are still discrepancies.

I'd only put a minimal example here, using the same privileges as documented for the writer role, and add a note that this assumes previous setup and link to the privileges docs for more details on that. Otherwise I am certain we will diverge again. 

I did not explicitly test the privileges again, but remember that we did extensive testing when overhauling the privileges docs, so I think we can rely on them.

Reply:
> I'd only put a minimal example here, using the same privileges as documented for the writer role, and add a note that this assumes previous setup and link to the privileges docs for more details on that. Otherwise I am certain we will diverge again.

Makes sense! I'll get that fixed.

> but remember that we did extensive testing when overhauling the privileges docs, so I think we can rely on them.

💯

Thanks!